packages/translate/lib/__tests__/__snapshots__/testLineResultThenExit.test.tsx.snap

```diff
@@ -0,0 +1,482 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Translate assert lines with then exit translate "assert application has exited" render with fail result 1`] = `
+"✔ Assert: Application has exited then stop test
+  fail: Condition was not met"
+`;
+
+exports[`Translate assert lines with then exit translate "assert application has exited" render with success result 1`] = `
+"» Assert: Application has exited then stop test
+  exit: Condition was met"
+`;
+
+exports[`Translate assert lines with then exit translate "assert application has exited" render without result 1`] = `"Assert: Application has exited then stop test"`;
+
+exports[`Translate assert lines with then exit translate "assert cookie" existingSubject error 1`] = `
+"✔ Assert: cookie cookie-name does not exist then stop test
+  fail: Condition was not met"
```

Comment:
should say "success"
@AntonSBilan status text must always be same as icon in front of the test line.